nbdev/export.py

```diff
@@ -40,8 +40,27 @@ def black_format(cell, # Cell to format
         try: cell.source = _format_str(cell.source).strip()
         except: pass
 
-# %% ../nbs/api/04_export.ipynb 9
-def nb_export(nbname, lib_path=None, procs=black_format, debug=False, mod_maker=ModuleMaker, name=None):
+# %% ../nbs/api/04_export.ipynb 10
+# includes the newline, because calling .strip() would affect all cells.
+_magics_pattern = re.compile(r'^\s*(%%|%).*\n?', re.MULTILINE)
```

Comment:
The magic pattern in `clean_magics` (in the `processors` notebook) is a little different. It doesn't include the newline and relies on calling `.strip()`. That's a much easier to follow, but seems to impact the whole cell instead of just the spark line. 

When I used `r'^\s*(%%|%).*'` to test `nbdev_export --procs scrub_magics`, an unrelated notebook (with no magics!) was stripped of their trailing whitespace. Probably not a big deal, but I wanted to minimize the disruption here.